### PR TITLE
PAE-158 Regis Footer

### PR DESCRIPTION
**Pull request by alexjmpb**
## Description:
This PR adds the style of the footer of the Regis theme
## Screenshots:
Screenshot 2022-07-13 at 08-09-24 Your Platform Name Here
Screenshot 2022-07-13 at 08-10-17 Your Platform Name Here
## Testing:
- Check out to this branch.
- Enable Comprehensive theming: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html
- Compile assets, `paver update_assets --themes=pearson-regis-theme`
- Assign 'pearson-regis-theme' to a new site.
## Reviewers:
- [ ] @Squirrel18
- [ ] @Jacatove

### CODE REVIEW

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_footer.scss [hunk 1]
```diff
@@ -27,18 +29,10 @@ $main-font-size: 0.875em;
       margin-left: initial;
       margin-bottom: 60px;
     }
-
+
```
Comment (Squirrel18) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_footer.scss]: Unnecessary identation. @alexjmpb

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_variables.scss [hunk 2]
```diff
@@ -4,17 +4,18 @@
...
-$font-family-sans-serif: Open Sans;
+$font-family-sans-serif: 'Roboto';
 $font-family-serif: Playfair Display, 'Georgia', 'Times New Roman', serif;
 
 // Initialize the Open edX Bootstrap theme
 @import 'edx-bootstrap/sass/open-edx/theme';
 // Import Bootstrap layout definitions.
 @import 'bootstrap/scss/bootstrap-grid';
 @import url("https://fonts.googleapis.com/css?family=Playfair+Display");
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
```
Comment (Squirrel18) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_variables.scss]: Missing end of file line.

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-regis-theme/lms/templates/footer.html [hunk 3]
```diff
@@ -66,46 +49,21 @@ <h4>Empowering people to make progress in their lives through learning</h4>
...
       </div>
       <div class="col-md-4 col-12 mid-footer-container">
-        <h4>Students</h4>
-        <p><a href="https://www.pearson.com/pathways/learner-profile.html/account">My Learner Profile</a></p>
-        <p><a href="https://www.pearson.com/pathways/recommended-programs.html">Get Recommendations</a></p>
-        <p><a href="https://www.pearson.com/pathways/areas-work-study.html">View Areas of Study</a></p>
-        <p><a href="https://www.pearson.com/pathways/program-profiles.html">Degrees, Courses &amp; Certificates</a></p>
-        <p><a href="https://www.pearson.com/pathways/student-resources.html">Student Resources</a></p>
-        <p><a href="https://www.pearson.com/pathways/blog.html">Education news and trends</a></p>
+        <p><a href="https://online.regiscollege.edu/accreditation/">Accreditation</a></p>
```
Comment (Jacatove) [inline edx-platform/pearson-regis-theme/lms/templates/footer.html]: I see the social media links open in a new tab, what do you think about doing the same for these links?. @alexjmpb

**Review by Jacatove — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_variables.scss [hunk 4]
```diff
@@ -4,17 +4,18 @@
...
-$footer-background: #0d2b52;
-$footer-contrast: rgba(255,255,255,0.7);
+$footer-background: #242E33;
+$footer-contrast: #F2F2F2;
 $dark: #313131;
 $gray: #E6E6E6;
 
 // Theme fonts
-$font-family-sans-serif: Open Sans;
+$font-family-sans-serif: 'Roboto';
```
Comment (Squirrel18) [inline edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_variables.scss]:
```suggestion
$font-family-sans-serif: 'Roboto', sans-serif;
```

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

**Review by Jacatove — APPROVED** [review]

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**